extensions/smallrye-opentracing/deployment/src/test/java/io/quarkus/smallrye/opentracing/deployment/NestedMdcScopesTest.java

```diff
@@ -0,0 +1,118 @@
+package io.quarkus.smallrye.opentracing.deployment;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import java.util.Map;
+
+import org.jboss.logging.MDC;
+import org.junit.jupiter.api.Test;
+
+import io.jaegertracing.internal.JaegerSpanContext;
+import io.opentracing.Scope;
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import io.opentracing.util.ThreadLocalScopeManager;
+import io.quarkus.jaeger.runtime.MDCScopeManager;
+
+public class NestedMdcScopesTest {
```

Comment:
Could be this test moved to Jaeger artifact - close to the MDC implementation?

Reply:
Hmm, I'm not sure why I added it to `smallrye-opentracing`, indeed this should be in `jaeger`. Thanks :-)

Reply:
Done.